paddle/infrt/dialect/infrt/infrt_ops.td

```diff
@@ -0,0 +1,52 @@
+#ifndef Infrt_OpS
+#define Infrt_OpS
+
+include "mlir/IR/OpBase.td"
+include "mlir/Interfaces/SideEffectInterfaces.td"
+
+def Infrt_Dialect : Dialect {
+  let summary =
+    "A dialect containing the Infrt Attributes, Operations, and Types";
+
+  let name = "Infrt";
+  let cppNamespace = "::infrt";
+}
+
+// Type definitions
+
+// Base class for Infrt dialect types.
+class Infrt_Type<string name, list<Trait> traits = [],
+                   string baseCppClass = "::mlir::Type">
+    : TypeDef<Infrt_Dialect, name, traits, baseCppClass> {
+}
+
+def LoDTensor : Infrt_Type<"LoDTensor"> {
```

Comment:
Currently the LoDTensor is just a DenseTensor without LOD operations?
Is it necessary to expose such concept to IR ? @Shixiaowei02